app/src/test/java/org/oppia/android/app/testing/options/AudioLanguageFragmentTest.kt

```diff
@@ -243,7 +243,7 @@ class DefaultAudioFragmentTest {
         .build() as TestApplicationComponent
     }
 
```

Comment:
```
private val component: TestApplicationComponent by lazy {
private val component: TestApplicationComponent by lazy {
      DaggerDefaultAudioFragmentTest_TestApplicationComponent.builder()
        .setApplication(this)
        .build() as TestApplicationComponent
    }
```
I think this snippet is what is causing the error.  
Becuase we have renamed the test file from `DefaultAudioFragmentTest` to `AudioLanguageFragmentTest`, the generated file `DaggerDefaultAudioFragmentTest_TestApplicationComponent` will no longer exist. Instead, another file called `DaggerAudioLanguageFragmentTest_TestApplicationComponent` will be generated.  
So, you need to change `DaggerDefaultAudioFragmentTest_TestApplicationComponent` to `DaggerAudioLanguageFragmentTest_TestApplicationComponent`

Reply:
I have tried this solution and tried to rebuild and invalidate the caches but still the dagger test component is not identified by Android studio.

Reply:
It may appear as `unresolved` in android studio although it is correct. So, to test that, we need to run the test cases in this file and observe whether they passed or failed.   

Also one more thing, the comment owner should resolve the comment by himself, so, please don't resolve the comments yourself. The reason behind that is that the reviewer uses these comments to track the PR.

Reply:
@ARJUPTA please check this link: [
Tips for getting your PR merged after submission](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#tips-for-getting-your-pr-merged-after-submission)

Reply:
Pardon me for that mistake, I didn't know about that. Should I squash the new commit and make a PR

Reply:
> Pardon me @MohamedMedhat1998 for that mistake,

No issues, it is totally fine :smiley: 



> Should I squash the new commit and make a PR?

Just push them and they will appear in this PR.